### DISCUSSION TITLE
local database

### DISCUSSION THREAD

**Discussion by NixFull** [invertase/react-native-firebase#6074] [Q&A]
Hello,

Is it possible to use firebase locally offline without any config on firebase console?
like realm DB, SQL Lite or WatermelonDB?

**Comment by mikehardy** [comment]
Hi there! No, it is not. Firebase is definitely a "cloud-first" library. That said, it is possible to begin development locally by using the firebase emulator suite. It just won't be possible to usefully deploy that to users

**Reply by NixFull** [reply]
Thanks for your answer
